src/content/blog/2023-02-03-community-gardening.md

```diff
@@ -0,0 +1,135 @@
+Since being approved for the grant, I've been better able to execute the role
+of community gardener; let me tell you about some of the ways in which the
+foundation's grant has enabled me to better support you, the GraphQL community!
+
+The [main GraphQL working group](https://github.com/graphql/graphql-wg) meets
+each month; and it is recorded on Zoom and made available via [the Foundation's
+YouTube
+channel](https://www.youtube.com/channel/UCERcwLeheOXp_u61jEXxHMA/playlists).
+Soon after each meeting I've been uploading the videos not just for the main
+meeting but also for the various other subcommittees (GraphiQL WG, GraphQL.js
+WG, GraphQL-over-HTTP WG, Metadata WG, Composite Schemas WG) that may have met
+during the previous month. Whilst uploading the video of the main meeting, I
+file its notes and the action items, while also categorizing them making sure
+they're assigned to the relevant person. Recently, I've also started adding
+timestamped links to the YouTube video for each action item, so we can hear the
+original instructions rather than just the paraphrased summaries from the
+notes - I think this makes them even more useful!
+
+## Empowering other contributors
+
+As part of my aim to document our processes and streamline things, I've created
+[a template repository](https://github.com/graphql/wg-template) that can be
+used whenever a new GraphQL subcommittee is created. This enables all
+subcommittees to adopt the GraphQL meeting practices with minimal effort from
+the committee chair - just [a small list of
+edits](https://github.com/graphql/wg-template#working-group-template) that take
+about 5 minutes when setting up the repository and of course the regular
+scheduling.
+
+I've also been working with the Linux Foundation's Elisa Trevino to get
+subcommittee meetings scheduled onto [the GraphQL
+calendar](https://calendar.google.com/calendar/embed?src=linuxfoundation.org_ik79t9uuj2p32i3r203dgv5mo8%40group.calendar.google.com),
+and to expand [EasyCLA](https://docs.linuxfoundation.org/v2/easycla) to the new
+repositories to ensure that everyone contributing has signed the [GraphQL
+Membership
+Agreement](https://github.com/graphql/graphql-wg/tree/main/membership). This
+should make releasing future versions of the spec and any subcommittee
+deliverables much easier since the intellectual property paperwork is in place
+from the start.
+
+Thanks to the new funding I've also been able to be more active on GitHub
+issues, PRs, discussions and in [the GraphQL Discord
+server](https://discord.graphql.org/), enabling me to offer assistance to
+people interested in contributing, and to help moderate these environments.
+This includes [explaining how to become a
+champion](https://github.com/graphql/graphql-spec/issues/174#issuecomment-1129621529),
+performing timely moderation actions when the Discord server suffers from spam,
+and giving guidance on how to improve PRs.
+
+## Extra-curricular activities
```

Comment:
You might want to also call out your awesome/massive work on the `struct` proposal! 😅

Reply:
I've gotta leave something in my pocket for future blog posts! 🤣